deployment/CP02PMUI_Deploy.csv

```diff
@@ -125,12 +125,12 @@ AR39,,AR44,,CP02PMUI-WFP01-02-DOFSTK000,14,1,2019-10-06T18:26:00,2020-06-13T16:0
-AR44,,,,CP02PMUI-SBS01-01-MOPAK0000,15,1,2020-06-13T13:47:00,,CGMCP-02PMUI-00015,,CGINS-MOPAK0-99999,40.36495,-70.780717,,0,94,
-AR44,,,,CP02PMUI-WFP01-00-WFPENG000,15,1,2020-06-13T13:47:00,,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGCON-CWFPCP-93601,40.36495,-70.780717,,70,94,
-AR44,,,,CP02PMUI-WFP01-01-VEL3DK000,15,1,2020-06-13T13:47:00,,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-VEL3DK-00022,40.36495,-70.780717,,70,94,
-AR44,,,,CP02PMUI-WFP01-02-DOFSTK000,15,1,2020-06-13T13:47:00,,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-DOFSTK-02397,40.36495,-70.780717,,70,94,
-AR44,,,,CP02PMUI-WFP01-03-CTDPFK000,15,1,2020-06-13T13:47:00,,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-CTDPFK-00106,40.36495,-70.780717,,70,94,
-AR44,,,,CP02PMUI-WFP01-04-FLORTK000,15,1,2020-06-13T13:47:00,,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-FLORTK-00969,40.36495,-70.780717,,70,94,
-AR44,,,,CP02PMUI-WFP01-05-PARADK000,15,1,2020-06-13T13:47:00,,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-PARADK-20423,40.36495,-70.780717,,70,94,
+AR44,,,,CP02PMUI-RII01-02-ADCPTG010,15,1,2020-06-13T13:47:00,2020-12-08T06:30:00,CGMCP-02PMUI-00015,,CGINS-ADCPTG-18596,40.36495,-70.780717,,71,94,Mooring was hit and riser fell to the seafloor
+AR44,,SI20201212,,CP02PMUI-SBS01-00-STCENG000,15,1,2020-06-13T13:47:00,2020-12-08T06:30:00,CGMCP-02PMUI-00015,,CGCON-BSTC01-00007,40.36495,-70.780717,,0,94,Mooring was hit and buoy went adrift
+AR44,,SI20201212,,CP02PMUI-SBS01-01-MOPAK0000,15,1,2020-06-13T13:47:00,2020-12-08T06:30:00,CGMCP-02PMUI-00015,,CGINS-MOPAK0-99999,40.36495,-70.780717,,0,94,Mooring was hit and buoy went adrift
+AR44,,,,CP02PMUI-WFP01-00-WFPENG000,15,1,2020-06-13T13:47:00,2020-12-08T06:30:00,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGCON-CWFPCP-93601,40.36495,-70.780717,,70,94,Mooring was hit and riser fell to the seafloor
+AR44,,,,CP02PMUI-WFP01-01-VEL3DK000,15,1,2020-06-13T13:47:00,2020-12-08T06:30:00,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-VEL3DK-00022,40.36495,-70.780717,,70,94,Mooring was hit and riser fell to the seafloor
+AR44,,,,CP02PMUI-WFP01-02-DOFSTK000,15,1,2020-06-13T13:47:00,2020-12-08T06:30:00,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-DOFSTK-02397,40.36495,-70.780717,,70,94,Mooring was hit and riser fell to the seafloor
+AR44,,,,CP02PMUI-WFP01-03-CTDPFK000,15,1,2020-06-13T13:47:00,2020-12-08T06:30:00,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-CTDPFK-00106,40.36495,-70.780717,,70,94,Mooring was hit and riser fell to the seafloor
+AR44,,,,CP02PMUI-WFP01-04-FLORTK000,15,1,2020-06-13T13:47:00,2020-12-08T06:30:00,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-FLORTK-00969,40.36495,-70.780717,,70,94,Mooring was hit and riser fell to the seafloor
+AR44,,,,CP02PMUI-WFP01-05-PARADK000,15,1,2020-06-13T13:47:00,2020-12-08T06:30:00,CGMCP-02PMUI-00015,CGVEH-CWFPCP-93601,CGINS-PARADK-20423,40.36495,-70.780717,,70,94,Mooring was hit and riser fell to the seafloor
```

Comment:
Verified that stopDateTime is the same as the last transmission time in FAR 311-00055. Recovery CUID looks good for buoy as well.

Reply:
looks good